services/telemetry/telemetry.go

```diff
@@ -331,6 +332,14 @@ func (ts *TelemetryService) trackActivity() {
 		activeUsersMonthlyCount = r.Data.(int64)
 	}
 
+	outgoingWebhooksCount, _ = ts.dbStore.Webhook().AnalyticsOutgoingCount("")
+
+	if license := ts.srv.License(); license != nil && license.Features.Cloud != nil && *license.Features.Cloud {
+		if usage, err := ts.dbStore.FileInfo().GetStorageUsage(true, false); err == nil {
+			storageBytes = utils.RoundOffToZeroes(float64(usage))
+		}
+	}
```

Comment:
Can we extend the tests at `telemetry_test.go` to include these cloud/non-cloud cases?